routers/repo/issue.go

```diff
@@ -52,8 +52,6 @@ const (
 )
 
 var (
-	// ErrTooManyFiles upload too many files
-	ErrTooManyFiles = errors.New("Maximum number of files to upload exceeded")
```

Comment:
Is there no linter check for unused variables?

Reply:
@silverwind the liners we use do not provide this feature - we would have to add a third one specific for this task ... the one I tested are not accurate :/

Reply:
It is about **Exported** unused variables, that a linter can not really decide if needed or not

Reply:
Ah yes in such a case the linter needs to be smarter. We actually do have such a feature in the JS linter that checks exported stuff too 😉